nav2_bringup/bringup/launch/tb3_simulation_launch.py

```diff
@@ -141,6 +165,8 @@ def generate_launch_description():
         cwd=[launch_dir], output='screen')
 
     urdf = os.path.join(bringup_dir, 'urdf', 'turtlebot3_waffle.urdf')
+    with open(urdf, 'r') as infp:
```

Comment:
Is this necessary? It looks before like we just gave `urdf` as an argument to the robot state publisher and it just handled it itself? Just asking the question